src/components/messages/__tests__/BadgeMessage.spec.ts

```diff
@@ -0,0 +1,32 @@
+import {beforeEach, test, expect, describe} from '@jest/globals'
+import {mount} from '@vue/test-utils'
+import BadgeMessage from '@/components/messages/BadgeMessage.vue'
+
+describe('BadgeMessage.vue', () => {
+  let wrapper: any
```

Comment:
Type should be `VueWrapper`

```suggestion
  let wrapper: VueWrapper
```

Reply:
@Aviv-Galmidi All fixed and pushed.